hoodie-common/src/main/java/com/uber/hoodie/common/util/ParquetUtils.java

```diff
@@ -52,6 +54,7 @@
      */
     public static Set<String> readRowKeysFromParquet(Path filePath) {
         Configuration conf = new Configuration();
+        conf.addResource(getFs().getConf());
```

Comment:
To use DFS for testing, the conf object needs to be passed in from the initialized file system. Instead of replacing the conf() from the filesystem, I add it as a resource.